code/examples/angular-cli/src/stories/addons/interactions/addon-interactions.stories.ts

```diff
@@ -66,16 +66,19 @@ Submitted.play = async (context) => {
   const canvas = within(context.canvasElement);
   await userEvent.click(canvas.getByText('Submit'));
 
-  await waitFor(async () => {
-    await expect(
-      canvas.getByRole('heading', {
-        name: /you submitted the following:/i,
-      })
-    ).toBeInTheDocument();
-    await expect(canvas.getByTestId('hero-name').textContent).toEqual('Storm');
-    await expect(canvas.getByTestId('hero-alterego').textContent).toEqual('Ororo Munroe');
-    await expect(canvas.getByTestId('hero-power').textContent).toEqual('Weather Changer');
-  });
+  await waitFor(
+    async () => {
+      await expect(
+        canvas.getByRole('heading', {
+          name: /you submitted the following:/i,
+        })
+      ).toBeInTheDocument();
+      await expect(canvas.getByTestId('hero-name').textContent).toEqual('Storm');
+      await expect(canvas.getByTestId('hero-alterego').textContent).toEqual('Ororo Munroe');
+      await expect(canvas.getByTestId('hero-power').textContent).toEqual('Weather Changer');
+    },
+    { timeout: 2000 }
+  );
```

Comment:
Idiomatically, this whole `waitFor` block should just be replaced with:

```js
await canvas.findByRole('heading', { name: /you submitted the following:/i }, { timeout: 2000 });
await expect(canvas.getByTestId('hero-name').textContent).toEqual('Storm');
await expect(canvas.getByTestId('hero-alterego').textContent).toEqual('Ororo Munroe');
await expect(canvas.getByTestId('hero-power').textContent).toEqual('Weather Changer');
```

Reply:
Agreed, but it was actually useful to highlight this. Having said that, all these stories are getting deleted in the next couple weeks and reworked into generic stories, in this case in https://github.com/storybookjs/storybook/pull/19153